galite-core/src/main/kotlin/org/kopi/galite/visual/RootItem.kt

```diff
@@ -67,41 +91,10 @@ class RootItem(id: Int, name: String) {
       self
     }
   }
-  // ---------------------------------------------------------------------
-  // ACCESSORS
-  // ---------------------------------------------------------------------
+
   /**
    * Returns true if this root item does not contain any item.
-   * @return True if this root item does not contain any item.
    */
   val isEmpty: Boolean
-    get() = this.rootNode == null
-
-  // ----------------------------------------------------------------------
-  // DATA MEMBERS
-  // ----------------------------------------------------------------------
-  /**
-   * Creates a root item from its ID and name.
-   * @param id The root item ID.
-   * @param name The root item name.
-   */
-  private val rootItem: Item = Item(id,
-          0,
-          name,
-          null,
-          null,
-          false,
-          false,
-          null,
-          name)
-
-  init {
-    rootItem.level = 0
-  }
-
-  /**
-   * Return the root node
-   */
-  var rootNode: TreeNode? = null
-    private set
+    get() = this.root == null
```

Comment:
Remove this keyword (  used when we need to distinguish between property and argument with same name which is not the case )  , also move properties with init to the end of file